lxd/instance/drivers/driver_qemu.go

```diff
@@ -2195,7 +2195,16 @@ func (d *qemu) deviceAttachPath(deviceName string, configCopy map[string]string,
 		return fmt.Errorf("Failed to connect to QMP monitor: %w", err)
 	}
 
```

Comment:
Keeping in mind that commits and their messages can be backported out of context of their original PR, messages like "Applying similar fix to ..." are not too helpful to the person backporting/reading them.

Please can you keep the commit messages descriptive to the specific logical change being made.

Thanks

Reply:
thanks for the insight, will keep that in mind